types/jest/index.d.ts

```diff
@@ -651,6 +651,11 @@ declare namespace jest {
          * Used to check that a JavaScript object matches a subset of the properties of an object
          */
         toMatchObject(expected: {} | any[]): R;
+        /**
+         * This ensures that a value matches the most recent snapshot with property matchers.
+         * Check out [the Snapshot Testing guide](http://facebook.github.io/jest/docs/snapshot-testing.html) for more information.
+         */
+        toMatchSnapshot<T extends {[P in keyof R]: Expect['any']}>(propertyMatchers: Partial<T>, snapshotName?: string): R;
```

Comment:
Why is an `Expect['any']` here? I think what is that saying is that the values of a property matcher can only be `expect.any()` calls. Is that correct? This works for other asymmetric matchers just because `Expect['any']` is of type `(a: any) => any`.